env-filter is currently not packaged due to version trouble, drop this patch when it's back

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -116,7 +116,6 @@
 features = [
     "fmt",
     "ansi",
-    "env-filter",
     "time",
     "tracing-log",
 ]
